src/netaddress.cpp

```diff
@@ -542,7 +542,7 @@ static std::string IPv4ToString(Span<const uint8_t> a)
     return strprintf("%u.%u.%u.%u", a[0], a[1], a[2], a[3]);
 }
 
-static std::string IPv6ToString(Span<const uint8_t> a)
+static std::string IPv6ToString(Span<const uint8_t> a, uint32_t scope_id)
```

Comment:
21985: incomplete, must wait for 21756 backport to be merged (via #6008)

Reply:
6008 is merged